netgrasp/config/config.py

```diff
@@ -1,16 +1,15 @@
 from netgrasp.utils import debug
 
 import ConfigParser
-import sys
 
 config_instance = None
 
-class Config:
-    def __init__(self, debugger):
+class Config(object):
```

Comment:
Adding support for Python3 is on my @TODO list. It's not something I've looked into yet, but something I hope to dive into shortly, time permitting.